src/main/java/uk/gov/hmcts/reform/civil/notification/handlers/notifyjudgmentvarieddeterminationofmeans/JudgmentVariedDeterminationOfMeansClaimantEmailDTOGenerator.java

```diff
@@ -0,0 +1,105 @@
+
+    private final NotificationsProperties notificationsProperties;
+    private final OrganisationService organisationService;
+    private final NotificationsSignatureConfiguration signatureConfig;
+    private final FeatureToggleService featureToggleService;
+
+    public JudgmentVariedDeterminationOfMeansClaimantEmailDTOGenerator(
+            NotificationsProperties notificationsProperties,
+            OrganisationService organisationService,
+            NotificationsSignatureConfiguration signatureConfig,
+            FeatureToggleService featureToggleService
+    ) {
+        this.notificationsProperties = notificationsProperties;
+        this.organisationService = organisationService;
+        this.signatureConfig = signatureConfig;
+        this.featureToggleService = featureToggleService;
+    }
+
+    @Override
+    protected Boolean getShouldNotify(CaseData caseData) {
+        String recipient = caseData.isApplicantLiP()
+                ? caseData.getApplicant1Email()
+                : caseData.getApplicantSolicitor1UserDetails().getEmail();
+        return nonNull(recipient);
+    }
+
+    @Override
+    protected String getEmailAddress(CaseData caseData) {
+        return caseData.isApplicantLiP()
+                ? caseData.getApplicant1Email()
+                : caseData.getApplicantSolicitor1UserDetails().getEmail();
+    }
+
+    @Override
+    protected String getEmailTemplateId(CaseData caseData) {
+        if (caseData.isApplicantLiP()) {
+            return caseData.isClaimantBilingual()
+                    ? notificationsProperties.getNotifyLipUpdateTemplateBilingual()
+                    : notificationsProperties.getNotifyLipUpdateTemplate();
+        }
+        return notificationsProperties.getNotifyClaimantJudgmentVariedDeterminationOfMeansTemplate();
+    }
+
+    @Override
+    protected String getReferenceTemplate() {
+        return REFERENCE_TEMPLATE;
+    }
+
+    @Override
+    public Map<String, String> addProperties(CaseData caseData) {
+        if (caseData.isApplicantLiP()) {
+            return Map.of(
+                    CLAIMANT_V_DEFENDANT,   getAllPartyNames(caseData),
+                    CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference(),
+                    PARTY_NAME,             caseData.getApplicant1().getPartyName()
+            );
+        }
+
+        Map<String, String> props = new HashMap<>();
+        props.put(CLAIM_REFERENCE_NUMBER, caseData.getCcdCaseReference().toString());
+        props.put(LEGAL_ORG_NAME,         getApplicantLegalOrganizationName(caseData, organisationService));
+        props.put(DEFENDANT_NAME,         getDefendantNameBasedOnCaseType(caseData));
+        props.put(PARTY_REFERENCES,       buildPartiesReferencesEmailSubject(caseData));
+        props.put(CASEMAN_REF,            caseData.getLegacyCaseReference());
+
+        addCommonFooterSignature(props, signatureConfig);
```

Comment:
this is not needed from our side.. it is already captured in base class email dto generator